analyzer/src/test/kotlin/curation/ClearlyDefinedPackageCurationProviderTest.kt

```diff
@@ -57,17 +57,20 @@ class ClearlyDefinedPackageCurationProviderTest : WordSpec({
             val provider = ClearlyDefinedPackageCurationProvider(Server.DEVELOPMENT)
 
             val identifier = Identifier("NPM:@nestjs:platform-express:6.2.3")
-            val curations = provider.getCurationsFor(identifier)
+            val curations = provider.getCurationsFor(listOf(identifier))
 
             curations should haveSize(1)
-            curations.first().data.concludedLicense shouldBe "Apache-1.0".toSpdx()
+
+            // TODO: Find out why the server returns "Apache-1.0" here for a GET (single) request, but "Apache-2.0" for
+            //       a POST (bulk) request.
```

Comment:
I'm actually surprised that it returns anything because I could not find any curations for that package here:
https://github.com/clearlydefined/curated-data/tree/master/curations/npm/npmjs

Also, the declared license for the package is clearly MIT so both Apache-1.0 and Apache-2.0 are wrong:
https://www.npmjs.com/package/@nestjs/platform-express/v/6.2.3
https://github.com/nestjs/nest/blob/v6.2.3/package.json

Or is the answer that the development server returns arbitrary values?

Reply:
> Or is the answer that the development server returns arbitrary values?

Yes 😕

Reply:
Ok, but then we don't need the TODO, right?

Reply:
The TODO is gone in the latest version in favor of a link to the issue that I filed meanwhile with ClearlyDefined (after checking in their chat that this actually *is* a bug).